packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -533,4 +533,43 @@ export default class BackupNg {
 
     return backupsByVmByRemote
   }
+
+  async checkVmBackupNg(id, srId, settings) {
+    const restoredId = await this.importVmBackupNg(id, srId, settings)
```

Comment:
The `try` should start right after this line to ensure nothing can prevent this VM from being destroyed at the end.